### ISSUE TITLE
Quality Control / Testing / Enhancements for Gravity Spy (and beyond)

### ISSUE THREAD

**Issue by alexbfree** [zooniverse/Panoptes-Front-End#2782]
Adding to this list as I find more.
Need to review all events from https://docs.google.com/document/d/1TTNgzAQbhYH9kn3mqcy0p_UBRIzFG3CgPxcXpTVv3Nk/edit#heading=h.a71ajjlkx95

We need to make sure `relatedID` and/or `data` are filled in, otherwise some of the events become meaningless. Also need to test all events logging correctly and capturing all details that make sense

Add to this list as we find more
- [ ] ensure pagination events capture the numbers of pages going to/from as well as sub type
- [ ] ensure view-post / like-post and similar post- or thread-specific events capture post, thread ID etc to `data` and maybe also `relatedID` where it makes sense 
- [ ] ensure details of username entered are captured when an anonymous user logs a login event.
- [ ] ...

I think it makes most sense for myself @alexbfree to do this (indeed part of this is what i anticipated for task 2 in https://github.com/zooniverse/Panoptes-Front-End/issues/2647), but since I this would be further down my to-do list, I will also add Jim and Will as assignees in case one of them gets to it before I do.

**Comment by srallen** [comment]
Gravity Spy is getting closer to a tentative launch, hopefully end of August. Does this still need to be done?

**Comment by alexbfree** [comment]
@srallen Just discussed with @trouille: Yes it absolutely does, I still have concerns about the quality and coverage of the logged data. Planning to do this early next week (w/c 5 Sep)

**Comment by alexbfree** [comment]
Hi, just wanted to update that I discussed my priorities with @chrislintott and he confirmed that I shouldn't be working on this until after I have finished work on the Volcrowe/Comet Hunters experiment as it is top priority to get that out ASAP - So I have not had chance to get to this yet. Hopefully Friday or Monday.

I guess the beta will just have to launch as is, if it can't be delayed (sorry, I don't want to be a bottleneck but I think it's inevitable).

In the meantime I'd suggest that if somebody wants to advance this, they could take a geordi dump and take a look at what data is being collected for Gravity Spy - this will help us to identify the shortfalls more quickly before I get to it.
